assets/js/components/ExecutionResults/ExecutionResults.jsx

```diff
@@ -159,9 +181,23 @@ function ExecutionResults({
         hosts={hosts}
         onContentRefresh={onContentRefresh}
         onStartExecution={onStartExecution}
+        selectedCheck={selectedCheck}
```

Comment:
The `ResultsContainer` doesn't have any `selectedCheck` field. 
I think you can remove this addition safely